### PR TITLE
New AID implemented.

### PR DESCRIPTION
**Pull request by dgrove727**

### CODE REVIEW

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import org.sleuthkit.autopsy.coreutils.Logger;
+
+@ActionID(category = "Tools", id = "org.sleuthkit.autopsy.experimental.autoingest.AutoIngestDashboardOpenAction")
+@ActionReference(path = "Menu/Tools", position = 104)
+@ActionRegistration(displayName = "#CTL_AutoIngestDashboardOpenAction", lazy = false)
+@Messages({"CTL_AutoIngestDashboardOpenAction=Auto Ingest Dashboard"})
+public final class AutoIngestDashboardOpenAction extends CallableSystemAction implements Presenter.Toolbar {
+    
+    private static final Logger LOGGER = Logger.getLogger(AutoIngestDashboardOpenAction.class.getName());
+    private static final String VIEW_IMAGES_VIDEOS = Bundle.CTL_AutoIngestDashboardOpenAction();
```
Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java]: I think "VIEW_IMAGES_VIDEOS" is left over from a copy paste.

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        UserPreferences.SelectedMode mode = UserPreferences.getMode();
+        return (mode == REVIEW);
+    }
+    
+    /** Returns the toolbar component of this action
+     *
+     * @return component the toolbar button */
+    @Override
+    public Component getToolbarPresenter() {
+        ImageIcon icon = new ImageIcon(getClass().getResource("btn_icon_image_gallery_26.png")); //NON-NLS
```
Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java]: Again this reference to image_gallery looks like a copy paste left over.

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    
+    private final JButton toolbarButton = new JButton();
+    private final PropertyChangeListener pcl;
+    
+    public AutoIngestDashboardOpenAction() {
+        super();
+        toolbarButton.addActionListener(actionEvent -> performAction());
+        pcl = (PropertyChangeEvent evt) -> {
+            if (evt.getPropertyName().equals(Case.Events.CURRENT_CASE.toString())) {
+                 setEnabled(RuntimeProperties.runningWithGUI() && evt.getNewValue() != null);
```
Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java]: is supposed to only be enabled while a case is open?

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties [hunk 4]
```diff
@@ -298,3 +298,38 @@ AutoIngestCasePanel.bnOpen.text=&Open
...
+AutoIngestControlPanel.pendingTable.toolTipText=The Pending table displays the order upcoming Jobs will be processed with the top of the list first
+AutoIngestControlPanel.bnPrioritizeCase.toolTipText=Move all images associated with a case to top of Pending queue.
+AutoIngestControlPanel.bnPrioritizeCase.text=Prioriti&ze Case
+AutoIngestControlPanel.bnPause.toolTipText=Suspend processing of Pending Jobs
+AutoIngestControlPanel.bnPause.text=Pause
+AutoIngestControlPanel.bnShowProgress.toolTipText=Show the progress of the currently running Job. This functionality is only available for jobs running on current AIM node.
+AutoIngestControlPanel.bnShowProgress.text=Ingest Progress
+AutoIngestControlPanel.bnOptions.toolTipText=Display options panel. All processing must be paused to open the options panel.
+AutoIngestControlPanel.bnOptions.text=&Options
+AutoIngestControlPanel.bnExit.toolTipText=Exit Application
```
Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties]: what are all the "&"s in this file?

Comment (dgrove727) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties]: I didn't put those in, but from my Visual C++ days, I believe the ampersand just underlines the character that follows and makes the corresponding keyboard key a shortcut to select the object. So, you can press 'O' to select the "Options" button.

Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties]: sounds resonable.

**Comment by dgrove727** [comment]
I went back to the class you requested some cleanup on and found that half of the code actually wasn't even necessary. So I took out some stuff, did a quick test again to make sure things were still operating as expected, and then pushed an updated version. So I believe this change should address your concerns.

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    private static final String DISPLAY_NAME = Bundle.CTL_AutoIngestDashboardOpenAction();
+    
+    @Override
+    public boolean isEnabled() {
+        UserPreferences.SelectedMode mode = UserPreferences.getMode();
+        return (mode == REVIEW);
+    }
+    
+    @Override
+    @SuppressWarnings("fallthrough")
```
Comment (millmanorama) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboardOpenAction.java]: I think suppressing the "fallthrough" warning is left over and dead.  It should be removed.

**Pull request merged**